app/assets/stylesheets/db/controllers/edit-requests.scss

```diff
@@ -0,0 +1,44 @@
+body.db-edit_requests-index {
+  table {
+    td.title-user {
+      .title {
+        margin-bottom: 3px;
+      }
+
+      .user {
+        a {
+          color: #9a9a9a;
```

Comment:
Color literals like `#9a9a9a` should only be used in variable declarations; they should be referred to via variable everywhere else.